src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -454,6 +455,7 @@ project-with-cluster:
             - 80
         ec2:
             cluster-size: 2
+            dns-external-primary: True
```

Comment:
I think `True` should be `true`

Reply:
I don't think it matters

Reply:
Afaik YAML is very forgiving and will accept all sorts of casing for booleans, but it would be useful to have a single all-lowercase convention so that `grep`, find/replace and other tools work consistently.

Reply:
> grep, find/replace and other tools work consistently.

that's kind of a weak reason ;) 

"I don't think it matters" == "I don't really care" , it's the sort of minutia teams get fixated on when there aren't more pressing problems. Perhaps what we actually need is an elife 3.0?